### ISSUE TITLE
Module fails WCAG AA standard

### ISSUE THREAD

**Issue by NightJar** [silverstripe/silverstripe-login-forms#135]
As this is the default log in page for many government departments and perhaps other sites around the world with legislation that demands AA compliance, this is a problem.

At a quick (non-definitive) scan:

WCAG 2.1 failures: 

- 1.4.1 Use of colour – The forgot password link relies on colour only to indicate it is a link. 
- 1.4.10 Reflow
  - The page header disappears and is not able to be read. A sighted user cannot ascertain what site they are logging in to. 
  - Registering with the authenticator app has a broken layout with instructions displaying outside the viewport (Figure 1) 
- 2.4.7 Focus visible – The focus indicator is provided by the browser only. In some browsers this is not very distinguishable. 

Figure 1:
![image](https://user-images.githubusercontent.com/778003/228495832-3b9e3311-24ae-431b-8b35-f9c7b8da3b36.png)
